public/app/features/logs/components/LogRows.tsx

```diff
@@ -142,7 +145,7 @@ class UnThemedLogRows extends PureComponent<Props, State> {
     // React profiler becomes unusable if we pass all rows to all rows and their labels, using getter instead
     const getRows = this.makeGetRows(orderedRows);
 
-    const getLogRowProperties = (row: LogRowModel) => {
+    const getLogRowProperties = (row: LogRowModel): ComponentProps<typeof LogRow> => {
```

Comment:
I've been thinking about this change ever since I saw it, because it will return a fresh object on every instance and I'm not sure about the impact on unnecessary re-renders.

Reply:
Ohh, right - good catch! Let me do a new PR to remove that.

Reply:
https://github.com/grafana/grafana/pull/70839